test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -180,3 +180,45 @@ Describe "Validate that Get-Help returns provider-specific help" -Tags @('CI', '
         }
     }
 }
+
+Describe "Validate about_help.txt under culture specific folder works" -Tags @('CI') {
+    BeforeAll {
+        $modulePath = "$pshome\Modules\Test"
```

Comment:
Please clarify - is "Module" hardcoded for searching help files? And do this folder always exist?